src/compressed_tensors/quantization/lifecycle/calibration.py

```diff
@@ -28,6 +28,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+# TODO: will be removed from compressed-tensors
```

Comment:
Would a deprecation warning be nicer? That way users would know this function will be removed w/o looking at the code

Reply:
Agree with @rahul-tuli on this. Ideally, I'd like to ensure we're not landing TODOs in he code and instead log or error as markers for the TODOs so it's more informative to the user and more likely to not be forgotten about